docs/modules/ROOT/pages/how-tos/exoscale/replace_storage_node.adoc

```diff
@@ -0,0 +1,547 @@
+====
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster patch cephcluster cluster --type=json \
+  -p '[{
+    "op": "replace",
+    "path": "/spec/healthCheck/daemonHealth/mon",
+    "value": {
+      "disabled": false,
+      "interval": "10s",
+      "timeout": "10s"
+    }
+  }]'
+kubectl --as=cluster-admin -n syn-rook-ceph-operator delete pods \
+  -l app=rook-ceph-operator
+----
+
+. Wait for operator to settle.
+Wait for a log message saying `done reconciling ceph cluster in namespace "syn-rook-ceph-cluster"`
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-operator logs -f \
+  deploy/rook-ceph-operator
+----
+
+. Cordon node to replace and delete MON pod
++
+[source,bash]
+----
+kubectl --as=cluster-admin cordon "${NODE_TO_REPLACE}"
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete pod \
+  -l app=rook-ceph-mon,ceph_daemon_id="${MON_ID}"
+----
+
+. Wait until new MON is scheduled
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pods -w
+----
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster exec -it deploy/rook-ceph-tools -- \
+  ceph status
+----
+
+. Verify that three MONs are running
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster get deploy -l app=rook-ceph-mon
+----
+
+. Reset the MON failover timeout
++
+[NOTE]
+====
+We currently have to restart the operator to force it to pick up the new MON health check configuration.
+Depending on the outcome from https://github.com/rook/rook/issues/8363[this issue] on GitHub, the operator restart may not be needed in the future.
+====
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster patch cephcluster cluster --type=json \
+  -p '[{
+    "op": "replace",
+    "path": "/spec/healthCheck/daemonHealth/mon",
+    "value": {}
+  }]'
+kubectl --as=cluster-admin -n syn-rook-ceph-operator delete pods \
+  -l app=rook-ceph-operator
+----
+
+=== Clean up the old node
+
+. Drain the node to replace
++
+[source,bash]
+----
+kubectl --as=cluster-admin drain "${NODE_TO_REPLACE}" \
+  --delete-emptydir-data --ignore-daemonsets
+----
+
+. Delete the node to replace from the cluster
++
+[source,bash]
+----
+kubectl --as=cluster-admin delete node "${NODE_TO_REPLACE}"
+----
+
+. Find the Exoscale node id of the node to replace
++
+[source,bash]
+----
+node_id=$(exo vm list -O json | \
+  jq --arg storage_node "$NODE_TO_REPLACE" -r \
+  '.[] | select(.name==$storage_node) | .id')
+----
+
+. Verify that the node ID is correct
++
+[source,bash]
+----
+exo vm list | grep "${node_id}"
+----
+
+. Delete the node
++
+[source,bash]
+----
+exo vm delete "${node_id}"
+----
+
+. Clean up localstorage PV of decommissioned node
++
+[source,bash]
+----
+kubectl --as=cluster-admin delete pv "${pv_name}"
+----
+
+=== Finish up
+
+. Re-enable ArgoCD auto sync
++
```

Comment:
There doesn't seem to be a step that says to disable ArgoCD sync?

Reply:
It doesn't say "Disable ArgoCD sync", but "Disable auto sync for component `rook-ceph`": https://github.com/appuio/openshift4-docs/pull/98/files#diff-20fce70a1a58209ea5e6297cd08cbbcf33acac367617ce7cb9670539bba0ab1aR203

Reply:
ah, missed that one and I was doing a browser search for "ArgoCD" :)